content/docs/1.55/deployment.md

```diff
@@ -279,116 +279,7 @@ docker run \
   jaegertracing/all-in-one:{{< currentVersion >}}
 ```
 
-#### Upgrade Badger v1 to v3
```

Comment:
make this change in `content/docs/latest/deployment.md`, not this file

Reply:
I didn't find the `latest/` folder, which contains the latest version. Do you mean `next-release/`?

Reply:
yes